railties/lib/rails/generators/app_base.rb

```diff
@@ -317,7 +328,7 @@ def assets_gemfile_entry
       end
 
       def webpacker_gemfile_entry
-        return [] if options[:skip_javascript]
```

Comment:
The webpack files were tied to the skip_javascript flag, which is happening in a few places. That wording is not right, so I changed it.